Example/ViewController+MiniAppMessageProtocol.swift

```diff
@@ -25,6 +26,17 @@ extension ViewController: MiniAppMessageProtocol, CLLocationManagerDelegate {
         }
     }
 
+    func requestCustomPermissions(permissions: [MASDKCustomPermissionModel], completionHandler: @escaping (Result<[MASDKCustomPermissionModel], Error>) -> Void) {
+        if let viewController = UIStoryboard(name: "Main", bundle: nil).instantiateViewController(withIdentifier: "CustomPermissionsTableViewController") as? CustomPermissionsTableViewController {
+            viewController.customPermissionHandlerObj = completionHandler
+            viewController.permissionsRequestList = permissions
```

Comment:
One of the requirements is `If perms have already been granted, the user shouldn't need to grant them again.`

So I think this means it that if the user requests a list of permissions which contains both granted and not yet granted permissions, then the user should not be shown the already accepted permissions again. So here you should filter out the permissions which have already been granted before passing to the ViewController so that it only displays toggles for the permissions which have not yet been granted. Do you think that makes sense?

Reply:
We have already taken care of it in SDK side, [User already responded list](https://github.com/rakutentech/ios-miniapp/pull/129/files#diff-ee09dac8c84e5da89672c0c27e4f6401R26) and [User not responded list](https://github.com/rakutentech/ios-miniapp/pull/129/files#diff-ee09dac8c84e5da89672c0c27e4f6401R30)